### DISCUSSION TITLE
Command class [ss] does not exist.

### DISCUSSION THREAD

**Discussion by ghost** [telegram-bot-sdk/telegram-bot-sdk#67] [Q&A] [answered]
Hello. Faced the following problem. My bot has a predefined list of commands. But, if I write a command that is not in the list, I get the error "Command class [ss] does not exist.". This breaks the logic of the bot, because after this error it no longer works for the user, but only floods the logs.
I think the solution to this problem will be to fork the library to your account, but maybe there is another solution? Thank you.

**Comment by irazasyed** [comment, marked as answer]
It's designed to fallback to help command, don't you have that in your registered commands?

You can customize that command and respond to the user as well as handle gracefully BTS.
